### DISCUSSION TITLE
failed get objs: failed to list objs: json: cannot unmarshal number 2755523652316298479 into Go struct field ShareFile.data.list.cid of type int

### DISCUSSION THREAD

**Discussion by ThomasChooo** [AlistGo/alist#6703] [Q&A]
32位系统，armv7l，挂载115分享链接报的错，文件夹能看到，点进去文件夹看不到文件报的错，挂载自己的115网盘没问题，看报错是超过int范围了，能否修复这个问题？谢谢。
解决方法：
使用int64，它在32位系统中也是64位宽，可以存放更大的数值。
如果你需要处理的数值在int64范围之外，你可以使用math/big包中的Big类型，它可以处理任意大的整数。

**Comment by Simonzhang8** [comment]
这个是不是应该提到Issue里或者你会修的话直接放到PR里？

**Reply by ThomasChooo** [reply]
已经提到issues里了#6704，我自己不会修改，还得请大神
